get-started.html

```diff
@@ -39,10 +39,10 @@ <h2 id="choose">
         <div class="start-content">
           <div class="left-column">
             <h2 id="get">
-              Get Grin</h2>
-            <p>You can get Grin by accepting it as a payment for goods and services. You can also buy Grin on different exchanges.</p>
+              Get grin</h2>
+            <p>You can get grin by mining, buying on an exchange, or accepting it as payment. Just remember, not your keys not your coins.</p>
```

Comment:
Get `grin` or `grins`? I think the plural makes more sense here.

> Just remember, not your keys not your coins.

Not sure if this is the right place to mention that.

Reply:
Usually a plural `s` isn't used. bitcoin, eth, monero etc. But i pondered about it for a few minutes. If you think it fits then sure.

Agree with the 2nd comment.

Reply:
It's unclear whether it's:
1. `buy grin` which for me indicate the amount in a singular form
1. `buy grins` which for me indicate the amount in a plural form
1. `buy Grin` which for me is incorrect because it refers to the network in general

I think all of them are okay let's keep the `buy grin` for now...

Reply:
I suggest we define the use of grin like deer, you never know if it's singular or plural

Reply:
buy grin seems the most natural